src/components/tree-views-container.js

```diff
@@ -453,14 +453,15 @@ const TreeViewsContainer = () => {
 
     const updateDirectoryTreeAndContent = useCallback(
         (nodeId) => {
+            dispatch(setCurrentChildren(undefined));
```

Comment:
If a new study arrives in the current directory, all the content is updated. Maybe move this when we change directory.

Reply:
modified,